src/components/experiment-tracking/runs-list-card/runs-list-card.scss

```diff
@@ -85,15 +87,15 @@
 }
 
 .runs-list-card__bookmark {
-  fill: var(--color-default-alt);
+  fill: var(--color-run-list-bookmark);
   height: 24px;
   margin-left: auto;
-  opacity: 0.7;
-  transition: opacity 300ms ease;
+  opacity: 1;
+  transition: fill 300ms ease;
   width: 24px;
 
   &:hover {
-    opacity: 1;
+    fill: var(--color-default-alt);
   }
 
   &--stroke {
```

Comment:
Just make sure to update the transition on this property to `transition: fill 300ms ease;` :)